CMakeLists.txt

```diff
@@ -181,7 +181,9 @@ find_package(Tbb REQUIRED)
 find_package(Readline REQUIRED)
 find_package(Curses REQUIRED)
 find_package(SQLite3 REQUIRED)
-find_package(Boost REQUIRED COMPONENTS container date_time)
+# Old versions of CMake used the FindBoost module, which is deprecated because find_package can now find the upstream package directly.
```

Comment:
What exactly does "upstream package" mean here?

Comment can be written on two lines.

Reply:
See [here](https://cmake.org/cmake/help/latest/policy/CMP0167.html)